### ISSUE TITLE
set up the environment

### ISSUE THREAD

**Issue by nooralahzadeh** [mega002/qdmr-based-question-generation#1]
Hi,
Thanks for sharing the code and wonderful research paper.
I tried to set up the environment to reproduce the code. However, I got an error in first step regarding to the :
```
git+git://github.com/mega002/allennlp@self-v1.1.0rc4#egg=allennlp
git+git://github.com/allenai/allennlp-models@v1.1.0rc4#egg=allennlp-models
git+git://github.com/mega002/public-transformers@self-v3.1.0#egg=transformers
```
Could you verify the links to the GitHub codes are correct?
Thanks

**Comment by nooralahzadeh** [comment]
I installed `allennlp==1.1.0rc4` and `allennlp-models==1.1.0rc4`. However, I got the error on running : 
```
python run_scripts/predict.py \
--model models/QDMR_parser_BART.tar.gz \
--data drop_dev_qdmr_input.csv \
--output-file drop_dev_qmdr_predictions.jsonl \
--gpu 0
```
ERROR: `allennlp.common.checks.ConfigurationError: key "serialization_dir" is required at location "model."`

**Comment by nooralahzadeh** [comment]
I did the following steps and it worked for me:
1- Use the following command for `allennlp` installation:  
`Pip install allennlp-models==v1.1.0rc4`
`Pip freez | grep allennlp`
2- Remove the "serialization_dir" from the module that the error comes from!

**Comment by nooralahzadeh** [comment]
I did the following steps and it worked for me:
1- Use the following command for `allennlp` installation:  
`Pip install allennlp-models==v1.1.0rc4`
`Pip freez | grep allennlp`
2- Remove the "serialization_dir" from the module that the error comes from!

**Comment by maxuetao** [comment]
Hi, I have the same problem when running the code.
My solution is that you must download the corresponding file from the author's repo and install it, and you cannot install it with pip. 
`git+git://github.com/mega002/allennlp@self-v1.1.0rc4#egg=allennlp`
`git+git://github.com/allenai/allennlp-models@v1.1.0rc4#egg=allennlp-models`
`git+git://github.com/mega002/public-transformers@self-v3.1.0#egg=transformers`

After comparison, I found that these packages have been modified by the author, so if you use pip to install the official version, many errors will be reported.
Although the problem has been solved for a long time, I hope it can help future users.

> I installed `allennlp==1.1.0rc4` and `allennlp-models==1.1.0rc4`. However, I got the error on running :
> 
> ```
> python run_scripts/predict.py \
> --model models/QDMR_parser_BART.tar.gz \
> --data drop_dev_qdmr_input.csv \
> --output-file drop_dev_qmdr_predictions.jsonl \
> --gpu 0
> ```
> 
> ERROR: `allennlp.common.checks.ConfigurationError: key "serialization_dir" is required at location "model."`
